rpc/params/network.go

```diff
@@ -1291,12 +1291,6 @@ func FromNetworkSpaceInfos(allInfos network.SpaceInfos) SpaceInfos {
 		mappedSubnets := make([]SubnetV3, len(si.Subnets))
 		for j, subnetInfo := range si.Subnets {
 			var mappedFanInfo *FanConfigEntry
```

Comment:
You should be able to delete this line as well. We could remove `FanInfo` from `SubnetV3` it is ignored if it is empty. Thus removing another mention of fan. This won't be a breaking change as it's not guaranteed to be there because it can be nil.

Reply:
Done, and I updated the QA steps including the 3.6->4.0 migration which shouldn't be impacted by this change. Anyways, the fanconfigurer facades along with any other Fan reference will be removed from the facades and rpc structs in the following days.